opencti-platform/opencti-graphql/src/database/data-initialization.js

```diff
@@ -270,7 +279,16 @@ const createBasicRolesAndCapabilities = async (context) => {
       'SETTINGS_SETMARKINGS',
       'SETTINGS_SETLABELS',
     ],
-  });
+  };
+
+  if (isFeatureEnabled((PROTECT_SENSITIVE_CHANGES_FF))) {
+    connectorRoleInput = {
+      ...connectorRoleInput,
+      can_manage_sensitive_config: false
```

Comment:
I think it's not necessary anymore because for new role when feature flag is enabled the default is false now. But it can stay there, not a big deal.